content/en/docs/started/workstation/minikf-aws.md

```diff
@@ -0,0 +1,191 @@
++++
+title = "Deploy Kubeflow using MiniKF on AWS"
+description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via AWS"
+weight = 40
++++
+
+![MiniKF latest
+version](https://www.arrikto.com/wp-content/uploads/2020/11/minikf-latest-version-aws.svg
+"MiniKF latest version")
+
+This guide describes how to launch a MiniKF instance on AWS. MiniKF is a single
+VM solution on the AWS Marketplace, and installs:
+
+- Kubernetes (using Minikube)
+- Kubeflow
+- Kale, an orchestration and workflow tool for Kubeflow that enables you to run
+  complete data science workflows starting from a notebook
+  ([GitHub](https://github.com/kubeflow-kale))
+- Arrikto's Rok, a data management software for data versioning and
+  reproducibility
```

Comment:
Suggestions: 

- Added links to Minkube and Rok, rephrased a few things

```suggestion
This guide describes how to launch a MiniKF instance—a single
VM solution on the AWS Marketplace. Once deployed, MiniKF installs the following:

- Kubernetes (using [Minikube](https://minikube.sigs.k8s.io/docs/))
- Kubeflow
- [Kale](https://github.com/kubeflow-kale): an orchestration and workflow tool for Kubeflow that allows you to run
  complete data science workflows starting from a notebook
- Arrikto [Rok](https://docs.arrikto.com): a data management software for data versioning and
  reproducibility
```

Reply:
Will add links, thanks